Formula/v/verilator.rb

```diff
@@ -35,7 +35,7 @@ def install
     system "./configure", "--prefix=#{prefix}"
     ENV.deparallelize if OS.mac?
     # `make` and `make install` need to be separate for parallel builds
-    system "make"
+    system "make", "-j"
```

Comment:
`brew` sets `MAKEFLAGS` to build in parallel, so this is not needed (and will not respect user settings of `HOMEBREW_MAKE_JOBS`).